roles/ocm_remove_addon/vars/main.yaml

```diff
@@ -1,2 +1,2 @@
 ---
-wait_until_removed: false
+wait_until_removed: False
```

Comment:
You could use boolean IMO.
`wait_until_removed: false`
And use `bool` filter in cases when Ansible has trouble interpreting variables as boolean, e.g.
`when: "{{ wait_until_removed }} | bool"` although I don't think you need it in your code.